vita3k/gui/src/game_selector.cpp

```diff
@@ -251,7 +262,30 @@ void draw_game_selector(GuiState &gui, HostState &host) {
             ImGui::Selectable(game.title.c_str(), &selected[3], ImGuiSelectableFlags_SpanAllColumns, ImVec2(0, static_cast<float>(icon_size)));
             ImGui::NextColumn();
             if (std::find(std::begin(selected), std::end(selected), true) != std::end(selected)) {
-                gui.game_selector.selected_title_id = game.title_id;
+                if (host.cfg.show_live_area_screen) {
+                    live_area_game(gui, host);
+                } else
+                    gui.game_selector.selected_title_id = game.title_id;
+            }
+            if (host.io.title_id == game.title_id) {
+                static const auto BUTTON_SIZE = ImVec2(60.f, 0.f);
+                ImGui::PushStyleColor(ImGuiCol_Text, GUI_COLOR_TEXT_MENUBAR_OPTIONS);
+                if (ImGui::BeginPopupModal("Live Area Error", nullptr, ImGuiWindowFlags_AlwaysAutoResize)) {
+                    ImGui::PopStyleColor();
+                    ImGui::TextColored(GUI_COLOR_TEXT, "Failed open Livea Area for:\n%s '%s'.\nDo you want launch it?", game.title_id.c_str(), game.title.c_str());
```

Comment:
If you can't open live area, just launch it. No need for a popup I think.

Reply:
Hmm yes right

Reply:
after thinks, is needed, no forget i using scroll and arrow for switch in game list, so if i using like you said, it launch game also during this switch by accident

Reply:
Try to make it work I think.

Reply:
no, i said i can't, nbot works, this in used for everyy switch game, not only for click on it, so no want launch game by accident ^^